src/main/java/lotto/domain/prize/statistics/TotalPrizeAmount.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain.prize.statistics;
+
+import lotto.domain.prize.ranking.PrizeRankingCount;
+import lotto.exception.ErrorMessage;
+import lotto.exception.LottoException;
+
+public final class TotalPrizeAmount {
+    private final long amount;
+
+    public TotalPrizeAmount(final long amount) {
+        validatePositiveAmount(amount);
+        this.amount = amount;
+    }
+
+    private void validatePositiveAmount(final long amount) {
+        if (amount < 0) {
+            throw LottoException.of(ErrorMessage.INVALID_TOTAL_PRIZE_AMOUNT_PRICE_NEGATIVE);
+        }
+    }
+
+    public long getAmount() {
+        return amount;
+    }
+
+    public static TotalPrizeAmount from(final PrizeRankingCount prizeRankingCount) {
+        final long totalPrizeAmount = prizeRankingCount.getPrizeRankingCount().entrySet().stream()
+                .mapToLong(value -> {
+                    final long prizeAmount = value.getKey().getPrizeAmount().getAmount();
```

Comment:
이 부분은 디미터의 법칙을 위반했다고 볼 수도 있을 것 같아요..! getPrizeAmount 까지만 호출하고 getAmount는 PrizeAmount가 호출하게 나눠보는 것은 어떨까요?

Reply:
@skylar1220 동의합니다!! `PrizeRanking` 에서 `prizeAmount.getAmount()` 를 호출해주는 게 나아보입니다.
점(.)이 너무 많으면 의심해봐야겠어요...!! 🤨🤨🤨🤨
아래 처럼 고쳐봤어요!!

```java
// PrizeRanking.java

    public long getAmount() {
        return prizeAmount.getAmount();
    }
```

```suggestion
                    final long prizeAmount = value.getKey().getAmount();
```